package.json

```diff
@@ -2,10 +2,8 @@
   "name": "aicommits",
   "version": "1.0.2",
   "description": "Writes your git commit messages for you with AI",
-  "main": "bin/index.js",
-  "bin": {
-    "aicommits": "bin/index.js"
-  },
+  "files": ["bin"],
```

Comment:
Using `files` to select publish files will be more future proof than block-listing each new file via `.npmignore`